.eslintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+    "env": {
+        "browser": true,
+        "es6": false,
+        "jquery": true,
+        "greasemonkey": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:vue/essential",
+        "plugin:es5/no-es2016"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "vue",
+        "es5"
+    ],
+    "rules": {
+        "no-undef": 0,
+        "eqeqeq": "error",
+        "indent": ["warn", 2],
+        "brace-style": "error",
+        "keyword-spacing": ["error", { "before": true }],
+        "spaced-comment": ["error", "always"],
+        "quotes": ["error", "single", { "avoidEscape": true }],
+        "no-const-assign": "error",
```

Comment:
No need in es5, as it is anyway  handled like this:
```
Parsing error: The keyword 'const' is reserved [Error]
```

Reply:
removed